tests/benchmark/test_camera_ready_campaign.py

```
@@ -257,6 +262,229 @@ def test_load_campaign_config_rejects_malformed_synthetic_actuation_profile(
+        variability_sample={
+            "schema_version": "synthetic-actuation-variability-sample.v1",
+            "tuple_value": ("a", "b"),
+            "list_value": [1, None],
+        },
+    ).to_metadata()
+    assert tuple_metadata["variability_sample"]["tuple_value"] == ["a", "b"]
+    assert tuple_metadata["variability_sample"]["list_value"] == [1, None]
+
+    with pytest.raises(ValueError, match="sample_index must be >= 0"):
+        sample_synthetic_actuation_profile(base_profile, distribution, seed=17, sample_index=-1)
+    with pytest.raises(ValueError, match="variability_sample.schema_version"):
```

Ruff (0.15.17)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)